### DISCUSSION TITLE
About noembed size.

### DISCUSSION THREAD

**Discussion by 353157** [Alex-D/Trumbowyg#1386] [Q&A]
Hi,
The iframe is too small when insert the video from youtube.
How to adjust size of iframe after insert video without edit html manually?

Thank you.

**Comment by Alex-D** [comment]
It would be best to use CSS to set the width and height you want to keep the thing responsive.

**Reply by 353157** [reply]
I wondering where I could try to edit CSS and how to.
Appreciate for your help.

**Reply by Alex-D** [reply]
That's the role of the website CSS, not the editor

**Comment by 353157** [comment]
And it would be great if there is a function could do the iframe size adjustment without edit html manually for the normal user who don't know the html.
